packages/insomnia-cookies/src/cookies.ts

```diff
@@ -1,40 +1,38 @@
-const { CookieJar, Cookie } = require('tough-cookie');
+import { CookieJar, Cookie, CookieJSON } from 'tough-cookie';
 
 /**
  * Get a list of cookie objects from a request.jar()
- *
- * @param jar
  */
-module.exports.cookiesFromJar = function(jar) {
+export const cookiesFromJar = (cookieJar: CookieJar): Promise<CookieJSON[]> => {
+  console.log({ cookieJar });
   return new Promise(resolve => {
-    jar.store.getAllCookies((err, cookies) => {
+    cookieJar.store.getAllCookies((err, cookies) => {
       if (err) {
         console.warn('Failed to get cookies form jar', err);
         resolve([]);
       } else {
         // NOTE: Perform toJSON so we have a plain JS object instead of Cookie instance
-        resolve(cookies.map(c => c.toJSON()));
+        resolve(cookies.map(cookie => cookie.toJSON()));
       }
     });
   });
 };
 
 /**
  * Get a request.jar() from a list of cookie objects
- *
- * @param cookies
  */
-module.exports.jarFromCookies = function(cookies) {
-  let jar;
+export const jarFromCookies = (cookies: Cookie[]) => {
+  let jar: CookieJar;
 
   try {
-    // For some reason, fromJSON modifies `cookies`. Create a copy first
-    // just to be sure
+    // For some reason, fromJSON modifies `cookies`.
+    // Create a copy first just to be sure.
     const copy = JSON.stringify({ cookies });
     jar = CookieJar.fromJSON(copy);
+    console.log({ jar });
```

Comment:
Same here re: unprefixed log message

Reply:
this is in error from when I was debugging.  good catch, actually.  I'll remove soon.

Reply:
fixed in 8a9da26a0d5a67403282723bcc306ec9bd2ac0ab